src/App.tsx

```diff
@@ -1,61 +1,131 @@
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { FormEvent, useState } from 'react';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+type GetUserById = (todoId: number) => User | null;
+
+const getUserById: GetUserById = (todoId: number) => {
+  return usersFromServer.find(user => user.id === todoId) || null;
+};
+
+const getAllIds = (array: Todo[]) => {
+  const ids: number[] = [];
+
+  array.forEach(element => ids.push(element.id));
+
+  return ids;
+};
```

Comment:
[CHECKLIST-2]: This function is not necessary and could be simplified. You could use the `map` method directly on the array to get all ids. Example: `const allIds = array.map(element => element.id);`

Reply:
Good point! 🔥 
Do it